app/src/main/java/org/kiwix/kiwixmobile/main/MainActivity.java

```diff
@@ -1304,6 +1308,10 @@ private void updateBottomToolbarVisibility() {
   @Override
   protected void onNewIntent(Intent intent) {
     super.onNewIntent(intent);
+    boolean handled = handleIntent(intent);
```

Comment:
No need of this variable, this line can be merged with the next.

Reply:
Ok @abdulwd